src/main/scala/org/biodatageeks/sequila/pileup/converters/samtools/SamtoolsSchema.scala

```diff
@@ -0,0 +1,25 @@
+package org.biodatageeks.sequila.pileup.converters.samtools
+
+import org.apache.spark.sql.types._
+
+object SamtoolsSchema {
+    val contig = 0
+    val position = 1
+    val ref = 2
+    val cov = 3
+    val pileupString = 4
+    val qualityString = 5
+    val altsMap = 6
+    val qualsMap = 7
+
+    val schema: StructType = StructType(
+        List(
+            StructField("contig", StringType, nullable = true),
```

Comment:
Maybe we should use our project contants for column names ?

Reply:
done